### ISSUE TITLE
凉凉月色

### ISSUE THREAD

**Issue by rogercmq** [ml-cs385/Deep-Learning-and-Feature-Visualization#1]
### Week14 Mengqi: VAE模型的初始化与优化，GAN实现

- [ ] for example, Xavier

  > reference: https://www.cnblogs.com/hellcat/p/8496956.html
  >
  > 更靠谱的reference: https://blog.csdn.net/VictoriaW/article/details/72872036

- [ ] 现在是adam优化器

  > anything else

- [ ] loss function改进

  > reference: https://github.com/swasun/VQ-VAE-Images

- [x] pytorch dataloader (already reshaped)
  - [x] StanfordDog
  - [x] PASCAL VOC

- [ ] Run VAE model

  - [ ] CUB200
  - [ ] StanfordDog
  - [ ] Pascal VOC
  - [x] MNIST
  - [x] Cifar10 

- [ ] DCGAN, CGAN on Cifar10

  > reference: https://github.com/mafda/generative_adversarial_networks_101

**Comment by rogercmq** [comment]
截至第十四周周四：

- [ ] 参考了 https://github.com/a514514772/Pytorch-VAE-GAN 在VAE基础上又加了一层GAN 在Cifar10上获得的"生成样本"更加清晰 （纯VAE糊的一批）

- [ ] 对于几个目标检测数据集input img shape不同，导致无法批处理样本的问题，在image reconstruction任务中只需要resize(96,96)，然后按照之前的training flow做应该也可以。
